src/Uno.Extensions.Maui.UI/MauiHost.cs

```diff
@@ -97,6 +103,8 @@ public MauiHost()
 		ActualThemeChanged += OnActualThemeChanged;
 	}
 
+	public event EventHandler<MauiContentCreatedEventArgs> MauiContentCreated = delegate { };
```

Comment:
I'd suggest `MauiContentChanged` rather than Created to be more consistent with other framework events